hack/install.sh

```diff
@@ -237,7 +266,7 @@ do_install() {
 			exit 0
 			;;
 
-		fedora|centos)
+		fedora|centos|oracleserver)
```

Comment:
ya we kinda have tooo :(

mostly because it is generated in the rpm build from the dockerfiles

On Mon, Jul 27, 2015 at 8:04 PM, Avi Miller <notifications@github.com>
wrote:

> In hack/install.sh
> <https://github.com/docker/docker/pull/14927#discussion_r35610818>:
>
> > @@ -237,7 +266,7 @@ do_install() {
> >  			exit 0
> >  			;;
> >
> > -		fedora|centos)
> > +		fedora|centos|oracleserver)
>
> Except oracleserver is Oracle Linux's lsb_dist value. That's what is
> returned by lsb_release and what was assigned to us by LSB.
>
> If you must have it as oraclelinux, then you'll need to split out
> oracleserver as the lsb_dist and replace lsb_dist with oraclelinux.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/14927/files#r35610818>.
>